app/styles/_component-variables.scss

```diff
@@ -502,7 +502,7 @@ body {
 
   //variables for appmonitoring
   --appmonitoring-font-weight-medium: var(--font-weight-medium);
-  --appmonitoring-color-black: var(--common-black);
+  --appmonitoring-color-ios: var(--ios);
```

Comment:
To be consistent with android let's use `--appmonitoring-platform-ios-color: var(--ios);`